### PR TITLE
pseudo-greenlight-poll for WBTC-B

### PR DESCRIPTION
**Pull request by ultraschuppi**
### Attention (CC)

@LongForWisdom

## Summary of changes

added pseudo-greenlight-poll for adding a ETH-B similar vault-type for WBTC

## Relevant Context

followup of [forum signal](https://forum.makerdao.com/t/signal-request-new-iam-vault-type-for-wbtc-with-lower-lr/5736/3)

### CODE REVIEW

File: governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Should we add a new WBTC vault type with lower LR? - January 11, 2021
```
Comment (LongForWisdom) [inline governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md:2]: I would go with something like 'Investigate implementation of a WBTC-B vault-type with a lower LR', also I don't think titles can have '?' due to some weird thing on the portal.

Comment (LongForWisdom) [reply governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md:2]: I'd update the filename to match this as well.

File: governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+summary: This poll will provide guidance to the Domain Teams for creating a new WBTC based vault type
```
Comment (LongForWisdom) [inline governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md:3]: I think I would take this straight from the greenlight polls: 'Signal your support or opposition to prioritising onboarding a WBTC-B vault-type'

File: governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+discussion_link: https://forum.makerdao.com/t/signal-request-new-iam-vault-type-for-wbtc-with-lower-lr/5736/
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Should we add a new WBTC vault type with lower LR? - January 11, 2021
```
Comment (LongForWisdom) [inline governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md:11]: Likewise, match title and lose '?'

File: governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. The community can vote in this poll to express support or opposition to add a new WBTC-based vault type with a lower Liquidation Ratio - similar to ETH-B.
+
+This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, January 11th at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
+
+## Review
+
```
Comment (LongForWisdom) [inline governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md:20]: I'd add a little more detail here as to the similarity of this to greenlight. I'm happy to fill this in later though, so feel free to leave as is.

File: governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-new-iam-vault-type-for-wbtc-with-lower-lr/5736/) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following action will be taken:**
+
+The new vault-type gets added to the collateral prioritisation list for the risk team and other mandated actors to prioritise, evaluate and potentially implement when they feel it is the most valuable use of time.
```
Comment (LongForWisdom) [inline governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md:27]: 'A WBTC-B vault-type will be added to the collateral prioritisation list...'

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Thank you @ultraschuppi! Feel free to make as many of these as you want, I'll clean it up as necessary when I come to merge it and put it up.

**Comment by ultraschuppi** [comment]
@LongForWisdom thanks for commenting, tried my very best.

the reason i held of from using "WBTC-B" is that there is a parallel initiative driven by [B.Protocol](https://forum.makerdao.com/t/discussion-wbtc-b-collateral-type-backed-by-b-protocol-with-x10-leverage/5409) that also plans for WBTC-B...

**Pull request merged**